src/api/spec/cassettes/Webui_ProjectController/GET_create/with_a_param_called_maintenance_project/it_should_behave_like_a_valid_project_saved/1_20_2_2_2.yml

```diff
@@ -0,0 +1,36 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: http://backend:5352/source/home:tom:my_project/_project/_history?deleted=1
```

Comment:
I'm curious: what are all these _history calls about? it seems to happen everywhere